robottelo/ui/locators/model.py

```diff
@@ -252,7 +253,11 @@ def __repr__(self):
         if keys:
             msg.append("contains=%s" % str(keys))
```

Comment:
@abalakh @ldjebran 
 On personal projects I set encoding on files and make import unicode literals, so everything is unicode and you don't have to not bother with str issues:

```python
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

cont = 'contains=%s'
print cont % 'foo'
print(cont, type(cont))
s = cont % u'á'

print(s, type(s))```

Result:
```console
contains=foo
(u'contains=%s', <type 'unicode'>)
(u'contains=\xe1', <type 'unicode'>)
```

Even removing unicode literals, python 2 decode str to unicode for you:

```python
# -*- coding: utf-8 -*-

cont = 'contains=%s'
print cont % 'foo'
print(cont, type(cont))
s = cont % u'á'

print(s, type(s))```

But you can check str contains is not unicode, so we are mixing different types:
```console
contains=foo
('contains=%s', <type 'str'>)
(u'contains=\xe1', <type 'unicode'>)
```

Exception raises when using non ascii chars on a file not defining encoding:
```python
cont = 'contains=%s'
print cont % 'foo'
print(cont, type(cont))
s = cont % u'á'

print(s, type(s))```

But this case tests would fail:
```console
  File "/home/renzo/PycharmProjects/sandbox2.7/s.py", line 6
SyntaxError: Non-ASCII character '\xc3' in file /home/renzo/PycharmProjects/sandbox2.7/s.py on line 6, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

This is something we wouldn't worry anymore if moving completely to python3 dropping support to python 2. I hope  we can move soon to avoid it and some ugly code using 6 only to keep it compatible  ;)